serialization_test.go fails on expapi due to default values set for APIGroup.

```
ok  	k8s.io/kubernetes/pkg/admission	0.108s
--- FAIL: TestRoundTripTypes (1.15s)
	serialization_test.go:72: 1: ThirdPartyResourceList: diff: (*expapi.ThirdPartyResourceList){TypeMeta:(api.TypeMeta){Kind:(string) APIVersion:(string)} ListMeta:(api.ListMeta){SelfLink:(string)Uɖ(巒慣Ţ ResourceVersion:(string)5389403074538565592} Items:([]expapi.ThirdPartyResource)[{TypeMeta:(api.TypeMeta){Kind:(string) APIVersion:(string)} ObjectMeta:(api.ObjectMeta){Name:(string)芥KǬRɟɭy减禕ɋǇa繯OŰ羁撀E GenerateName:(string)睲ǒʠ巃fcE Namespace:(string) SelfLink:(string)B谔 UID:(types.UID):Ð辝Óȿʎó汯褐ȐYU掣4W唙e ResourceVersion:(string)17013635890666560228 Generation:(int64)0 CreationTimestamp:(util.Time){Time:(time.Time){sec:(int64)0 nsec:(int32)0 loc:(*time.Location)<nil>}} DeletionTimestamp:(*util.Time)<nil> DeletionGracePeriodSeconds:(*int64)<nil> Labels:(map[string]string)<nil> Annotations:(map[string]string)<nil>} Description:(string)ʭ汪ǻkʠ嚲ȶTÅ十>ȟy Versions:([]expapi.APIVersion)[{Name:(string)Kʞ"枟Ȫ_ǙQȞÔƖ APIGroup:(string)
		
		A: }]}]}
		
		B: experimental}]}]}
		
		
		Codec: &{0x4c2080da9a0 v1}
		Data: {"kind":"ThirdPartyResourceList","apiVersion":"v1","metadata":{"selfLink":"Uɖ(巒慣Ţ","resourceVersion":"5389403074538565592"},"items":[{"metadata":{"name":"芥KǬRɟɭy减禕ɋǇa繯OŰ羁撀E","generateName":"睲ǒʠ巃fcE","selfLink":"B谔","uid":":Ð辝Óȿʎó汯褐ȐYU掣4W唙e","resourceVersion":"17013635890666560228","creationTimestamp":null},"description":"ʭ汪ǻkʠ嚲ȶTÅ十\u003eȟy","versions":[{"name":"Kʞ\"枟Ȫ_ǙQȞÔƖ"}]}]}
		Source: "(*expapi.ThirdPartyResourceList){TypeMeta:(api.TypeMeta){Kind:(string) APIVersion:(string)} ListMeta:(api.ListMeta){SelfLink:(string)Uɖ(巒慣Ţ ResourceVersion:(string)5389403074538565592} Items:([]expapi.ThirdPartyResource)[{TypeMeta:(api.TypeMeta){Kind:(string) APIVersion:(string)} ObjectMeta:(api.ObjectMeta){Name:(string)芥KǬRɟɭy减禕ɋǇa繯OŰ羁撀E GenerateName:(string)睲ǒʠ巃fcE Namespace:(string) SelfLink:(string)B谔 UID:(types.UID):Ð辝Óȿʎó汯褐ȐYU掣4W唙e ResourceVersion:(string)17013635890666560228 Generation:(int64)0 CreationTimestamp:(util.Time){Time:(time.Time){sec:(int64)0 nsec:(int32)0 loc:(*time.Location)<nil>}} DeletionTimestamp:(*util.Time)<nil> DeletionGracePeriodSeconds:(*int64)<nil> Labels:(map[string]string)<nil> Annotations:(map[string]string)<nil>} Description:(string)ʭ汪ǻkʠ嚲ȶTÅ十>ȟy Versions:([]expapi.APIVersion)[{Name:(string)Kʞ\"枟Ȫ_ǙQȞÔƖ APIGroup:(string)}]}]}"
		Final: "(*expapi.ThirdPartyResourceList){TypeMeta:(api.TypeMeta){Kind:(string) APIVersion:(string)} ListMeta:(api.ListMeta){SelfLink:(string)Uɖ(巒慣Ţ ResourceVersion:(string)5389403074538565592} Items:([]expapi.ThirdPartyResource)[{TypeMeta:(api.TypeMeta){Kind:(string) APIVersion:(string)} ObjectMeta:(api.ObjectMeta){Name:(string)芥KǬRɟɭy减禕ɋǇa繯OŰ羁撀E GenerateName:(string)睲ǒʠ巃fcE Namespace:(string) SelfLink:(string)B谔 UID:(types.UID):Ð辝Óȿʎó汯褐ȐYU掣4W唙e ResourceVersion:(string)17013635890666560228 Generation:(int64)0 CreationTimestamp:(util.Time){Time:(time.Time){sec:(int64)0 nsec:(int32)0 loc:(*time.Location)<nil>}} DeletionTimestamp:(*util.Time)<nil> DeletionGracePeriodSeconds:(*int64)<nil> Labels:(map[string]string)<nil> Annotations:(map[string]string)<nil>} Description:(string)ʭ汪ǻkʠ嚲ȶTÅ十>ȟy Versions:([]expapi.APIVersion)[{Name:(string)Kʞ\"枟Ȫ_ǙQȞÔƖ APIGroup:(string)experimental}]}]}"
FAIL
FAIL	k8s.io/kubernetes/pkg/api	1.361s
```

--- a/pkg/api/serialization_test.go
+++ b/pkg/api/serialization_test.go
@@ -32,9 +32,8 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 	"k8s.io/kubernetes/pkg/util"
 
-	// TODO: enable when exapi problems are fixed #13083
-	//_ "k8s.io/kubernetes/pkg/expapi"
-	//_ "k8s.io/kubernetes/pkg/expapi/v1"
+	_ "k8s.io/kubernetes/pkg/expapi"
+	_ "k8s.io/kubernetes/pkg/expapi/v1"
 
 	flag "github.com/spf13/pflag"
 )
@@ -125,7 +124,7 @@ func TestList(t *testing.T) {
 	roundTripSame(t, item)
 }
 
-var nonRoundTrippableTypes = util.NewStringSet("ThirdPartyResource")
+var nonRoundTrippableTypes = util.NewStringSet()
 var nonInternalRoundTrippableTypes = util.NewStringSet("List", "ListOptions", "PodExecOptions", "PodAttachOptions")
 var nonRoundTrippableTypesByVersion = map[string][]string{}
 

--- a/pkg/api/testing/fuzzer.go
+++ b/pkg/api/testing/fuzzer.go
@@ -26,6 +26,7 @@ import (
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/registered"
 	"k8s.io/kubernetes/pkg/api/resource"
+	"k8s.io/kubernetes/pkg/expapi"
 	"k8s.io/kubernetes/pkg/fields"
 	"k8s.io/kubernetes/pkg/labels"
 	"k8s.io/kubernetes/pkg/runtime"
@@ -300,6 +301,11 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 		func(n *api.Node, c fuzz.Continue) {
 			c.FuzzNoCustom(n)
 			n.Spec.ExternalID = "external"
+		},
+		func(s *expapi.APIVersion, c fuzz.Continue) {
+			// We can't use c.RandString() here because it may generate empty
+			// string, which will cause tests failure.
+			s.APIGroup = "something"
 		},
 	)
 	return f

